src/common/entity/compute_active_media_states.ts

```diff
@@ -0,0 +1,28 @@
+import type { HassEntity } from "home-assistant-js-websocket";
+import type { HomeAssistant } from "../../types";
+import { generateEntityFilter } from "./entity_filter";
+
+export function computeActiveMediaStates(
```

Comment:
Can we move this to `/data/media-player.ts` to keep all media player related logic in one place?